addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5428,12 +5436,12 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13144"
-msgid "Change Apple Remote mode?"
+msgid "Change Apple remote mode?"
```

Comment:
I think this is wrong, the Apple Remote is a specific thing so it should be capitalized, not just a general Apple-named remote.